notebooks/01-Python-and-Notebooks.ipynb

```diff
@@ -0,0 +1,681 @@
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "- Python notebooks (files with `.ipynb` extension) are designed to work with an application called **Jupyter**"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- However, VSCode can be used instead, via a [*Jupyter* Extension](https://marketplace.visualstudio.com/items?itemName=ms-toolsai.jupyter) supported by Microsoft itself.\n",
+    "\n",
+    "  - If you're already familiar with VSCode, or planning to use Python beyond just notebooks, it's probably a better option than Jupyter."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- You'll need to have a Python environment set up for it to run the code in.\n",
+    "\n",
+    "  - We'll discuss more about Python environments later in the class."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "![Example Python Notebook](images/example-python-notebook.jpg)"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Course File Structure"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "- VSCode shows a file browser on the left pane (you may have to click the document icon).\n",
+    "\n",
+    "- If you open the folder containing the course files, you'll be able to see how it's organized: we have folders such as `scripts`, `notebooks`, and `.github`.\n",
+    "  - However, the only one that matters to you is `notebooks` -- that's where the content of the training lives.\n",
+    "\n",
+    "- These slides, for example, were created from the `notebooks/01-Python-and-Notebooks.ipynb` file."
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "slide"
+    }
+   },
+   "source": [
+    "### Python Notebooks"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* Notebooks allow the writing AND running of Python code"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {
+    "slideshow": {
+     "slide_type": "fragment"
+    }
+   },
+   "source": [
+    "* Notebooks are organized by **cells** - code and commentary text goes in the cells\n",
+    "  \n",
+    "  * A cell can contain *either* markdown (stylized text) or Python code."
+   ]
+  },
```

Comment:
This slide is a bit duplicative of the "Notebook Cells" slide - does it need to be here too?

(line 392)

Reply:
Good catch and I agree that it's redundant (also not that relevant to the very first slide on notebooks). Updated in 80cc0c9.

Reply:
I replaced it with a different, more important idea. Also fixed some inconsistent punctuation choices that I noticed.